docs/deploying_apps/deploying_rails.md

```diff
@@ -1,28 +1,44 @@
-See the official CF guide: [Getting Started Deploying Ruby on Rails Apps](http://docs.cloudfoundry.org/buildpacks/ruby/gsg-ror.html). Here are the minimal steps for a basic Rails app:
 
-1. Follow the setup instructions in [Getting Started](/overview/quick_start_guide/) and review [concepts](/overview/concepts/).  If you are just getting started learning CloudFoundry, you can use the sandbox space:
+For full details of how to deploy Ruby on Rails apps, see the official Cloud Foundry guide [Getting Started Deploying Ruby on Rails Apps](http://docs.cloudfoundry.org/buildpacks/ruby/gsg-ror.html). 
 
-    ```
-    cf target -o sandbox
-    ```
+Note that the only database service currently supported by PaaS is PostgreSQL. If your Rails app requires a database, it must be able to work with PostgreSQL (this should be the case if the app follows the 12-factor principle of treating the database as a loosely-coupled backing service).
```

Comment:
I don't think this is quite right. 12 factor apps must be able to work with a database in any location, supplied by any provider, but it's fine for it to require that database is MySQL - 12 factor doesn't specify that databases should comply to the same standard, or that apps should use adapter layers to be compatible with any database.